src/serac/infrastructure/accelerator.hpp

```diff
@@ -88,18 +90,33 @@ void terminateDevice();
 /**
  * @brief Utility method to display last cuda error message
  *
- * @param[in] o The output stream to post success or CUDA error messages
  * @param[in] success_string A string to print if there are no CUDA error messages
  */
-inline void displayLastCUDAErrorMessage(std::ostream& o, const char* success_string = "")
+inline void displayLastCUDAErrorMessage(const char* success_string = "")
 {
   auto error = cudaGetLastError();
   if (error != cudaError::cudaSuccess) {
-    o << "Last CUDA Error Message :" << cudaGetErrorString(error) << std::endl;
+    SLIC_ERROR_ROOT(serac::profiling::concat("Last CUDA Error Message :", cudaGetErrorString(error)));
```

Comment:
This will result in program termination (which is not the current behavior) - do we want this to just be a warning that doesn't immediately kill the program?

Reply:
Ahh. I can see how it might be useful both ways. What is the best SLIC "stream" to use for soft errors?

Reply:
you have INFO (for regular messages) , WARNING, ERROR. Only ERROR halts.